doc/Installation/Installation-Debian-10-Nginx.md

```diff
@@ -134,13 +134,15 @@ server {
 ```
 
 ```bash
-systemctl reload nginx
+ln -s /etc/nginx/sites-available/librenms.vhost /etc/nginx/sites-enabled/librenms.vhost
+service nginx reload
 ```
 
 # Configure snmpd
 
 ```bash
 cp /opt/librenms/snmpd.conf.example /etc/snmp/snmpd.conf
+chmod 600 /etc/snmp/snmpd.conf
```

Comment:
you change mod to 600 ... doesn't work if snmpd.conf is not owned by snmpd user

Reply:
if you limit access such high a chown own is also needed

Reply:
It is not. snmpd is started as root and privileges are dropped after starting. snmpd must be started as root to acquire privileged port 161.

snmpd.conf is read as root, before dropping privileges.